praetorian_cli/ui/console/commands/accounts.py

```
@@ -0,0 +1,260 @@
+            self.console.print('[dim]Run "accounts" first to see available accounts.[/dim]')
+            return
+
+        target = args[0]
+
+        # Allow numeric selection from accounts list
+        if target.isdigit() and hasattr(self, '_account_list'):
+            idx = int(target) - 1
+            if 0 <= idx < len(self._account_list):
+                target = self._account_list[idx]
+            else:
+                self.console.print(f'[error]Invalid number. Run "accounts" to see the list.[/error]')
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)